plugins/service-monitor-activemq/src/main/java/com/thinkbiganalytics/service/activemq/util/ActivemqPoolableConnectionProvider.java

```diff
@@ -0,0 +1,81 @@
+package com.thinkbiganalytics.service.activemq.util;
+
+/*-
+ * #%L
+ * service-monitor-activemq
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.jms.pool.PooledConnectionFactory;
+
+import org.apache.log4j.Logger;
+
+import com.thinkbiganalytics.service.activemq.config.ActivemqTransportListner;
+
+public class ActivemqPoolableConnectionProvider {
+
+
+    private static final Logger log = Logger.getLogger(ActivemqPoolableConnectionProvider.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "log" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)